tyk-pro/values.yaml

```diff
@@ -190,7 +194,7 @@ dash:
   # variables in extraEnvs object array to define your SSL cert and key files.
   tls: false
   # Dashboard license.
-  license: ""
+  license: "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhbGxvd2VkX25vZGVzIjoiMTJjZjc5Y2EtNWYyOS00N2FmLTY5OTQtZTYyMWFmMTExMGM1LDNiOTE1MzY5LWJlN2QtNDJlOS00YWEzLTUxNWFlZmUwNzNlYyxlY2MzNTQ5MS1hNmJlLTQ0NDMtNWMwMS0yOTVjYzZmZWM0ZWYsMDcwYTM2MTEtZWMzNi00MzI1LTRhZDAtNjA5ZDI5NGE1ODdhLGRmZjYyNDcyLWM4NjItNGE2Ny00N2RmLTE5MjQ2ODQxYjFmMCxmYTY1NjM1NS1lYzM0LTQyNWUtNjgxNC04Yzc5NjcxODdhZTEsZjE0NTFkOGMtMTVhOS00MWQwLTU4MzYtN2NhY2UyMzIyNjE0LGMyZDZhM2FiLTI5NzctNDM3MS02MDc5LWE5MWNhYTQwZDA5NSxiMzJmOWNmYi0xZTFmLTRkNjMtNDgxMi1lMWViZWY4ZGUyNWYsNGFiNzRiNTQtZTgyNy00M2RmLTU4ZjgtNmUzYzBiNGVhYzM3IiwiZXhwIjoxNjUwNjk3MzY3LCJpYXQiOjE2NDgxMDUzNjcsIm93bmVyIjoiNjIwZDM4ZTRhY2QwM2YwMDAxZGYzNTc1Iiwic2NvcGUiOiJtdWx0aV90ZWFtLHJiYWMsZ3JhcGgsZmVkZXJhdGlvbiIsInYiOiIyIn0.Qu7C7tVw9Z41IrbJJl1Eap0QDQ_K1dlOHzzAEVWZiupr_yt5Tt2lgbpIBU6CC-_DCopUWWZ5fh0x8IOG4y45c67Hw8vq2LfnEU956LONQmWuBDzA9K7yokwZqQOFOLgnNT8DVRvrAAVI0xfWimG-ZzlU2vtSrQUybTNGMTvFNXh0DTAo9en1OUq9Te9inBoBZ7b47ZNHBPx9PHLgmwHLS6P4A8w2Td1FtKvlJRov_c6W7lMiTdiSG8MjUwBORhTOMgyxJjgat7NAaxsNbms57KrULiD5iIFgnD1WT3T8iJMV_yCR5nBLI0dTUjUTsYDsDr5EdQOPza9_3RLfmagKcQ"
```

Comment:
I guess license needs be removed from here

Reply:
this is now removed